control/tests/test_statefbk.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python
 #
-# statefbk_test.py - test state feedback functions
+# test_statefbk.py - test state feedback functions
```

Comment:
Not sure why we are changing the name of this file.  All other files in this directory have `_test` as a suffix on the file name.  Suggest we leave unchanged.

Reply:
I saw in issue https://github.com/python-control/python-control/issues/95 that you plan to organise the unit tests as in numpy. In fact, in this library as well as in many others, tests are always prefixed with `test_`:
![image](https://user-images.githubusercontent.com/22080415/56080254-c9c94d00-5dfe-11e9-9578-47c09db2f317.png)

in this PR, as I refactored code related to state feedback, I also started to rename unit tests properly. Do you want me to make this change in a dedicated PR, for all other tests?

Reply:
Current practice in python-control is to (mainly) use `_test` at the end of the filename, so prefer to leave this as is (no need to change).